RtAudio.cpp

```diff
@@ -5301,26 +5370,35 @@ void RtApiWasapi::wasapiThread()
           convertBuffer( stream_.deviceBuffer,
                          stream_.userBuffer[OUTPUT],
                          stream_.convertInfo[OUTPUT] );
-
+          // Convert callback buffer to stream sample rate
+          renderResampler->Convert(convBuffer,
+                                    stream_.deviceBuffer,
+                                    stream_.bufferSize,
+                                    convBufferSize);
+          // Push callback buffer into outputBuffer
+          callbackPushed = renderBuffer.pushBuffer(convBuffer,
+                                                    convBufferSize * stream_.nDeviceChannels[OUTPUT],
+                                                    stream_.deviceFormat[OUTPUT]);
+        }
+        else if (renderResampler->get_sampleratio()==1) {
```

Comment:
This is not necessary as this check is done in renderResampler->Convert()

Reply:
The second commit is not intended, the second commit was trying to reduce a few copies when there's no need. I'm really sorry for this messy setup of pr.

Reply:
No worries, could you please made the necessary pushes to your branch such that only the required changes remain. Thanks!